app/uk/gov/hmrc/fileupload/frontendGlobal.scala

```diff
@@ -188,12 +188,10 @@ class ApplicationModule(context: Context) extends BuiltInComponentsFromContext(c
 
   lazy val scanner: () => AvScanIteratee = new VirusScanner(configuration, environment).scanIteratee
   lazy val scanBinaryData: (EnvelopeId, FileId, FileRefId) => Future[ScanResult] = {
-    val enableClam = configuration.getConfig(s"${environment.mode}.clam.antivirus").flatMap(_.getBoolean("enableScanning")).getOrElse(true)
-    if (!enableClam) {
-      (_: EnvelopeId, _: FileId, _: FileRefId) => Future.successful(Xor.right(ScanResultFileClean))
-    } else {
-      ScanningService.scanBinaryData(scanner, getFileFromQuarantine)(createS3Key)
-    }
+    val disableScanning = configuration.getConfig(s"${environment.mode}.clam.antivirus")
+                            .flatMap(_.getBoolean("disableScanning")).getOrElse(false)
+    if (disableScanning) (_: EnvelopeId, _: FileId, _: FileRefId) => Future.successful(Xor.right(ScanResultFileClean))
```

Comment:
I can see why you (Waldemar) have suggested this change but I think the previous name is better